bsip-00XX Sharedrop Operation.md

```diff
@@ -0,0 +1,153 @@
+    BSIP: TBD
+    Title: Sharedrop operation
+    Authors: OpenLedgerApp <https://github.com/OpenLedgerApp>
+    Status: Draft
+    Type: Protocol
+    Created: 2018-12-17
+    Updated: 2018-12-17
+    Discussion: 
+    Worker: TBD
+ 
+A sharedrop is a giveaway of some tokens to a certain group of people, for example to all BTS holders in proportion to their holdings.
+
+# Abstract
+We should add the ability to distribute arbitrary amount of some token (SHAREDROP token) to holders of a specific other token (STAKE token)  according to the share(stake) of such tokens.
+The importance of a similar solution was raised in BSIPs-19, 20,  however never implemented. (link below).
```

Comment:
This is pretty much the same thing as paying dividends.
The dividends feature has been implemented already for the PeerPlays blockchain.
See here: https://github.com/bitshares/bitshares-core/issues/695

Reply:
As I mentioned, the idea is very similar to bsip-20 (and 19).  
We have spent quite a lot of time looking into the PeerPlay code. And at this point of time (2 years after the code has been written) we have concluded that it is easier, safer and more maintainable in the future to write our own code. 
Please see my extensive reply here: 
https://github.com/bitshares/bsips/issues/123#issuecomment-444119802
Thanks,
Denis

Reply:
We have just posted related BSIP  - Market Fee Backed Asset (MFBA) - #134 
and here:
https://bitsharestalk.org/index.php?topic=27601.0
The idea is quite similar, but it's more automated with MFBA. 
And it's more flexible with Sharedrop. 
We can have one or the other or both. 
Code base, when it comes to splitting share with stakeholders to their vesting balances would be quite similar.

Reply:
What is the difference to BSIP 19/20 in essence?